src/utils/Table/Filters/TextSearchFilter.js

```diff
@@ -5,7 +5,7 @@ import { getFieldValue } from "../utils";
 import ResetableFilter from "./ResetableFilter";
 
 const TextSearchFilter = React.forwardRef(({
-    value, onChange, onCommitChange, className, label, setActiveFilters, id, column, placeholder,
+    value, onChange, onCommitChange, className, label, setActiveFilters, id, column, placeholder, isCaseSensitive = true,
```

Comment:
Maybe `isCaseSensitive` should always be true. I don't see the reason why we need case sensitive search.

Reply:
@TiagooGomess the comment is a bit contraditory. I believe the isCaseSensitive flag should be false as default (so that javascript matches Javascript, for example). I agree that we might not need case sensitive search

Reply:
What I meant was that isCaseSensitive should always be false, my bad.